lbz/exceptions.py

```diff
@@ -16,6 +16,9 @@ def __init__(self, message: str = ""):
         if message:
             self.message = message
 
+    def __str__(self) -> str:
+        return f"{self.status_code} {self.message}"
```

Comment:
This is a string that will be visible in logs, if you have a better idea, feel free to share it with me 🙂

Reply:
How about braces for status code?

`return f"[{self.status_code}] {self.message}"`

Reply:
Please adjust based on Rolands suggestion